src/library/scala/compat/Platform.scala

```diff
@@ -13,22 +13,23 @@
 package scala
 package compat
 
-import java.lang.System
-
+@deprecated("Will be removed in Scala 2.14.0.",since = "Scala 2.13.0")
```

Comment:
Shouldn't the since parameters be "2.13.0" instead of "Scala 2.13.0" ?

Reply:
Yes, my old deprecator tool would have choked. I'm not sure mechanical parsing is guaranteed, however. My complaint would have been to add a space after the comma. I think they added sorting the messages by the "since" field, which I'm not sure is useful. Has anyone ever made a change on the basis of that field?

Reply:
Probably after the calendar New Year is OK if it is important to you, or after Lunar New Year if that is important to you.

Reply:
> Probably after the calendar New Year is OK if it is important to you, or after Lunar New Year if that is important to you.

I can do it now:) previously it's the big sale of `12.12`, so I missed this, and last weekend I went back to hometown, So~~:)

Reply:
refs:https://github.com/scala/scala/pull/7577